include/oneapi/dpl/experimental/kt/esimd_radix_sort.h

```diff
@@ -40,63 +40,58 @@ __onesweep_process_size()
 }
 
 // TODO: allow calling it only for all_view (accessor) and guard_view (USM) ranges, views::subrange and sycl_iterator
-template <std::uint16_t WorkGroupSize, std::uint16_t DataPerWorkItem, bool IsAscending, std::uint32_t RadixBits,
-          typename _ExecutionPolicy, typename _Range,
-          oneapi::dpl::__internal::__enable_if_device_execution_policy<_ExecutionPolicy, int> = 0>
+template <bool _IsAscending, std::uint32_t _RadixBits, typename _KernelParam, typename _Range>
 void
-radix_sort(_ExecutionPolicy&& __exec, _Range&& __rng)
+radix_sort(sycl::queue __q, _Range&& __rng, _KernelParam __param)
```

Comment:
... and then we may use here it's instance: ````_KernelParam __param = {}```

Reply:
See my answer above.